src/main/scala/edu/berkeley/ce/rockslicing/NumericUtils.scala

```diff
@@ -39,11 +41,12 @@ object NumericUtils {
   /**
     * Rounds to the specified number of decimal places. If not specified, default
     * is 6 decimal places
+    *
     * @param d A Double value
     * @param decimals Number of decimal places to round to.
     * @return Value with specified number of decimal places. Default is 6 decimal places.
     */
-  def roundToTolerance(d: Double, decimals: Int=6): Double = {
-    BigDecimal(d).setScale(decimals, BigDecimal.RoundingMode.HALF_UP).toDouble
+  def roundToTolerance(d: Double, decimals: Int=7): Double = {
+      BigDecimal(d).setScale(decimals, BigDecimal.RoundingMode.HALF_DOWN).toDouble
```

Comment:
What's up with the change to `HALF_DOWN`? Isn't `HALF_UP` what most people are used to anyways?